src/Ledger/PParams.lagda

```diff
@@ -47,65 +46,60 @@ record PParams : Set where
 \begin{code}
-        maxBlockSize       : ℕ
-        maxTxSize          : ℕ
-        maxHeaderSize      : ℕ
-        maxValSize         : ℕ
-        pv                 : ProtVer -- retired, keep for now
+        maxBlockSize maxTxSize        : ℕ
+        maxHeaderSize maxValSize      : ℕ
+        pv                            : ProtVer -- retired, keep for now
 \end{code}
 \emph{Economic group}
 \AgdaTarget{a, b, minUTxOValue, poolDeposit}
 \begin{code}
-        a                  : ℕ
-        b                  : ℕ
-        minUTxOValue       : Coin
-        poolDeposit        : Coin
+        a b                           : ℕ
+        minUTxOValue poolDeposit      : Coin
 \end{code}
 \emph{Technical group}
 \AgdaTarget{Emax, collateralPercent}
 \begin{code}
-        Emax               : Epoch
-        collateralPercent  : ℕ
+        Emax                          : Epoch
+        collateralPercent             : ℕ
 \end{code}
 \emph{Governance group}
 \AgdaTarget{drepThresholds, poolThresholds, ccMinSize, ccMaxTermLength, govActionLifetime, govActionDeposit, drepDeposit, drepActivity, minimumAVS}
 \begin{code}
-        drepThresholds     : DrepThresholds
-        poolThresholds     : PoolThresholds
-        govActionLifetime  : ℕ
-        govActionDeposit   : Coin
-        drepDeposit        : Coin
-        drepActivity       : Epoch
-        ccMinSize          : ℕ
-        ccMaxTermLength    : ℕ
-        minimumAVS         : Coin
+        drepThresholds                : DrepThresholds
+        poolThresholds                : PoolThresholds
+        govActionLifetime             : ℕ
+        govActionDeposit drepDeposit  : Coin
+        drepActivity                  : Epoch
+        ccMinSize ccMaxTermLength     : ℕ
+        minimumAVS                    : Coin
 
-        -- Script
-        -- costmdls            : Language →/⇀ CostModel (Does not work with DecEq)
-        costmdls            : CostModel
-        prices              : Prices
-        maxTxExUnits        : ExUnits
-        maxBlockExUnits     : ExUnits
-        coinsPerUTxOWord    : Coin
-        -- collateralPercent   : ℕ
-        maxCollateralInputs : ℕ
+  -- Script group
+        -- costmdls                   : Language →/⇀ CostModel (Does not work with DecEq)
+        costmdls                      : CostModel
+        prices                        : Prices
+        maxTxExUnits                  : ExUnits
+        maxBlockExUnits               : ExUnits
+        coinsPerUTxOWord              : Coin
+        -- collateralPercent          : ℕ
+        maxCollateralInputs           : ℕ
 
 paramsWellFormed : PParams → Set
-paramsWellFormed pp = 0 ∉ fromList
-    ( maxBlockSize ∷ maxTxSize ∷ maxHeaderSize ∷ maxValSize ∷ minUTxOValue ∷ poolDeposit
-    ∷ collateralPercent ∷ ccMaxTermLength ∷ govActionLifetime ∷ govActionDeposit
-    ∷ drepDeposit ∷ [] )
-  × ℕtoEpoch govActionLifetime ≤ drepActivity
+paramsWellFormed pp =
```

Comment:
Imho, this formatting of the proof of `paramsWellFormed` is much nicer (and it would look even nicer to me if `×` were aligned with `0`, but that seems to be an unpopular style around here).